### DISCUSSION TITLE
Please help to convert table to chart bar

### DISCUSSION THREAD

**Discussion by cryptobandteam** [ChartsCSS/charts.css#75] [Q&A] [answered]
Hello. I am glad to find chartscss as a simplest way to convert table to chart.

I added chartscss css to my site and inserted table but my table did not convert to chart. Please help to fix this or explain what I am doing wrong

<table class="charts-css bar">
<tbody>
<tr>
<td style="text-align: right;">01.02.2022</td>
<td style="text-align: right;">$300</td>
</tr>
<tr>
<td style="text-align: right;">06.02.2022</td>
<td style="text-align: right;">$540</td>
</tr>
<tr>
<td style="text-align: right;">21.02.2022</td>
<td style="text-align: right;">$780</td>
</tr>
<tr>
<td style="text-align: right;">02.03.2022</td>
<td style="text-align: right;">$690</td>
</tr>
<tr>
<td style="text-align: right;">07.03.2022</td>
<td style="text-align: right;">$690</td>
</tr>
<tr>
<td style="text-align: right;">26.03.2022</td>
<td style="text-align: right;">$930</td>
</tr>
<tr>
<td style="text-align: right;">10.04.2022</td>
<td style="text-align: right;">$1&nbsp;170</td>
</tr>
<tr>
<td style="text-align: right;">19.07.2022</td>
<td style="text-align: right;">$1&nbsp;170</td>
</tr>
<tr>
<td style="text-align: right;">20.08.2022</td>
<td style="text-align: right;">$1&nbsp;080</td>
</tr>
<tr>
<td style="text-align: right;">10.09.2022</td>
<td style="text-align: right;">$1&nbsp;080</td>
</tr>
<tr>
<td style="text-align: right;">15.09.2022</td>
<td style="text-align: right;">$1&nbsp;320</td>
</tr>
<tr>
<td style="text-align: right;">29.10.2022</td>
<td style="text-align: right;">$1&nbsp;230</td>
</tr>
<tr>
<td style="text-align: right;">08.11.2022</td>
<td style="text-align: right;">$1&nbsp;470</td>
</tr>
<tr>
<td style="text-align: right;">14.12.2022</td>
<td style="text-align: right;">$1&nbsp;380</td>
</tr>
<tr>
<td style="text-align: right;">19.12.2022</td>
<td style="text-align: right;">$1&nbsp;290</td>
</tr>
<tr>
<td style="text-align: right;">11.01.2023</td>
<td style="text-align: right;">$1&nbsp;530</td>
</tr>
</tbody>
</table>

**Comment by ramiy** [comment, marked as answer]
@cryptobandteam

Use this table:

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/charts.css/dist/charts.min.css">

<table class="charts-css bar show-labels">
  <tbody>
    <tr>
      <th scope="row" scope="row">01.02.2022</th>
      <td style="--size: calc( 300 / 1530 )">$300</td>
    </tr>
    <tr>
      <th scope="row" scope="row">06.02.2022</th>
      <td style="--size: calc( 540 / 1530 )">$540</td>
    </tr>
    <tr>
      <th scope="row">21.02.2022</th>
      <td style="--size: calc( 780 / 1530 )">$780</td>
    </tr>
    <tr>
      <th scope="row">02.03.2022</th>
      <td style="--size: calc( 690 / 1530 )">$690</td>
    </tr>
    <tr>
      <th scope="row">07.03.2022</th>
      <td style="--size: calc( 690 / 1530 )">$690</td>
    </tr>
    <tr>
      <th scope="row">26.03.2022</th>
      <td style="--size: calc( 930 / 1530 )">$930</td>
    </tr>
    <tr>
      <th scope="row">10.04.2022</th>
      <td style="--size: calc( 1170 / 1530 )">$1&nbsp;170</td>
    </tr>
    <tr>
      <th scope="row">19.07.2022</th>
      <td style="--size: calc( 1170 / 1530 )">$1&nbsp;170</td>
    </tr>
    <tr>
      <th scope="row">20.08.2022</th>
      <td style="--size: calc( 1080 / 1530 )">$1&nbsp;080</td>
    </tr>
    <tr>
      <th scope="row">10.09.2022</th>
      <td style="--size: calc( 1080 / 1530 )">$1&nbsp;080</td>
    </tr>
    <tr>
      <th scope="row">15.09.2022</th>
      <td style="--size: calc( 1320 / 1530 )">$1&nbsp;320</td>
    </tr>
    <tr>
      <th scope="row">29.10.2022</th>
      <td style="--size: calc( 1230 / 1530 )">$1&nbsp;230</td>
    </tr>
    <tr>
      <th scope="row">08.11.2022</th>
      <td style="--size: calc( 1470 / 1530 )">$1&nbsp;470</td>
    </tr>
    <tr>
      <th scope="row">14.12.2022</th>
      <td style="--size: calc( 1380 / 1530 )">$1&nbsp;380</td>
    </tr>
    <tr>
      <th scope="row">19.12.2022</th>
      <td style="--size: calc( 1290 / 1530 )">$1&nbsp;290</td>
    </tr>
    <tr>
      <th scope="row">11.01.2023</th>
      <td style="--size: calc( 1530 / 1530 )">$1&nbsp;530</td>
    </tr>
  </tbody>
</table>
```

**Reply by cryptobandteam** [reply]
Thank you. So, I should manually specify each <td> style?

**Reply by ramiy** [reply]
yes
